src/server/verification/verificationAPI.js

```diff
@@ -247,27 +247,33 @@ const setup = (app: Router, verifier: VerificationAPI, storage: StorageAPI) => {
     wrapAsync(async (req, res, next) => {
       const log = req.log.child({ from: 'verificationAPI - verify/sendemail' })
       const { user, body } = req
+      let userData
       // log.info({ user, body })
       //merge user details for use by mautic
-      let userRec: UserRecord = _.defaults(body.user, user)
+      try {
+        userData = user.toJSON()
```

Comment:
this needs to be done at the source.
this issue will repeat everywhere we use results from mongodb

Reply:
done

Reply:
need to change it at the mongo adapter level